src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.EVP.Cipher.cs

```diff
@@ -66,5 +66,16 @@ internal static partial class Crypto
 
         [DllImport(Libraries.CryptoNative, EntryPoint = "CryptoNative_EvpDes3Cbc")]
         internal static extern IntPtr EvpDes3Cbc();
+
+        [DllImport(Interop.Libraries.CryptoNative, EntryPoint = "CryptoNative_EvpRc4")]
+        internal static extern IntPtr EvpRc4();
+
+        [DllImport(Interop.Libraries.CryptoNative, EntryPoint = "CryptoNative_EvpCipher")]
+        internal static unsafe extern int EvpCipher(
+            SafeEvpCipherCtxHandle ctx,
+            byte[] output,
+            byte* input,
```

Comment:
It's strange to me that input comes after output in the argument ordering.  Also, what's the reason one is a ```byte[]``` and the other is a ```byte*```, rather than both being the same type?